CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Enable remote write receiver.
+- Add prometheus remote write ingress boilerplate.
+- Add prometheus agent remote write secret boilerplate.
+- Generate prometheus remote write agent secret and config.
```

Comment:
Those must be rebased after merging #1000

Reply:
Please consider merging #1000  first.

Reply:
it makes me a little bit confused, as it already includes changes of #1000

Reply:
Because it's not possible to have them separate and this saves me time to wait for reviews 😁